### ISSUE TITLE
(Feature request) plugin manager

### ISSUE THREAD

**Issue by zhang-stephen** [zadam/trilium#3238]
### Describe feature

I have tried trilium for a week and I like it very much.

I think there should be a powerful plugin(or widget) manager, to define the standard of widget, to import third-party widgets, and to manage them for conveniently. The installation method for now, download the code and add them into notes manually, is too complexed to use.

**Comment by zhang-stephen** [comment]
well, these are old issues...

**Comment by sigaloid** [comment]
Yes, but there are explanations there as to why it hasn't been implemented yet and what the likelihood is of it being implemented in the future.

**Comment by zadam** [comment]
@sigaloid is correct, not much has changed. It might still come, but not soon.
